### DISCUSSION TITLE
Extensions Toolbar Menu

### DISCUSSION THREAD

**Discussion by Marek33** [Eloston/ungoogled-chromium#1294] [Ideas]
How can I re-add this flag _Extensions Toolbar Menu_ to the Chromium 87? Or can someone else, add it back? I'm not sure, if the **Ideas** is the right category, but I would like to have this _new/old_ feature back.

Also, thank you very much, for all the work, and other people, who are working on Windows versions, since this is the best Chromium browser!

**Comment by ghost** [comment]
If you talking about auto-hidden icons for extensions I agree. It's really annoying when they auto-hide, I wish there were an option to disable this. So all of the icons will be visible all the time.

**Comment by Marek33** [comment]
@v00v Yes, that one. Hopefully, if the Iridium will bring it back, the Ungoogled, can use it somehow: https://github.com/iridium-browser/tracker/issues/312
